blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/OrganizationFolderRunImpl.java

```diff
@@ -97,6 +96,21 @@ public Date getEndTime() {
     }
 
     @Override
+    public String getStartTimeString() {
```

Comment:
Ignore that comment, for some reason I mistook it as one of the model class. All good here:)